### ISSUE TITLE
CLI: msw init prompt information error

### ISSUE THREAD

**Issue by hehehai** [mswjs/msw#132]
## msw init prompt information error
```
$ npx msw init ./public

Initializing the Mock Service Worker at '......'
Failed to copy Service Worker. null
```
msw is executed successfully, the file is copied normally, but the failure message is prompted

## Environment

- `"msw": "^0.15.3"`

## fix
cli/init.js
```js
fs.copyFile(SERVICE_WORKER_BUILD_PATH, swDestFilepath, (error) => {
 invariant(typeof error === null, 'Failed to copy Service Worker. %s', error)

 ...
}
```
`error` If there is no error message, it is `null`
`typeof null => 'object'`

fix
```js
fs.copyFile(SERVICE_WORKER_BUILD_PATH, swDestFilepath, (error) => {
 invariant(error === null, 'Failed to copy Service Worker. %s', error)

 ...
}
```
or
```js
fs.copyFile(SERVICE_WORKER_BUILD_PATH, swDestFilepath, (error) => {
 invariant(!error, 'Failed to copy Service Worker. %s', error)

 ...
}
```

**Comment by kettanaito** [comment]
Hey, @hehehai. Thanks for reporting this. It seems I've screwed up the invariant on that copying function. Will need to look at it once more.

Usually invariant behaves that if the condition it receives is _truthy_, it throws an error.

**Comment by kettanaito** [comment]
@hehehai, would you please have a second to look at the pull request that should fix this? #133

**Comment by kettanaito** [comment]
Fix is released in `0.15.4`. Could you please update and let me know that successful copying of the worker file not doesn't produce the error message? Thanks.

**Comment by hehehai** [comment]
@kettanaito It's working
![image](https://user-images.githubusercontent.com/12692552/80895454-9f06ab00-8d17-11ea-9e5f-00d5faf75fb6.png)
